SCF-Session-4/clumps/ArrayClumps.java

```diff
@@ -0,0 +1,46 @@
+package clumps;
+
+/**
+ * Contains a method to calculate the number of clumps in an array.
+ * @author Ishita Mathur
+ *
+ */
+public class ArrayClumps 
+{
+	/**
+	 * This method returns the number of clumps in an array.
+	 * @param inputArray, should contain a series of 2 or more adjacent elements of the same value
+	 * @param arrayLength > 0
+	 * @return the count of clumps in the array.
+	 */
+	public int countClumps(int[] inputArray, int arrayLength)
```

Comment:
Why second parameter `arrayLength` is needed? Length can be calculated through `inputArray.length`.